echopype/convert/api.py

```diff
@@ -362,10 +370,23 @@ def open_raw(
         and need to be added to the converted file
     storage_options : dict
         options for cloud storage
+    offload_to_zarr: bool
+        If True, variables with a large memory footprint will be
+        written to a temporary zarr store.
+    max_zarr_mb : int
+        maximum MB that each zarr chunk should hold, when offloading
+        variables with a large memory footprint to a temporary zarr store
 
     Returns
     -------
     EchoData object
+
+    Notes
+    -----
+    If ``offload_to_zarr='auto'``, a heuristically driven method will
+    determine if variables with a large memory footprint should be
+    written to a temporary zarr store. This method is currently only
+    available for the following echosounders: EK60, ES70, EK80, ES80, EA640.
```

Comment:
Remove notes regarding to `auto` string.

Reply:
Dang, I missed that. Thank you for catching this!

Reply:
Hehe you have a lot of code to keep track of. That's what another pair of eyes are for 😉

Reply:
Absolutely!